python/pyarrow/table.pxi

```diff
@@ -1327,6 +1327,8 @@ cdef class Table(_PandasConvertible):
             Display Field-level and Schema-level KeyValueMetadata.
         preview_cols : int, default 0
             Display values of the columns for the first N columns.
+        cols_char_limit : int, default 200
```

Comment:
The default in the docstring does not matches the value in function arguments.

Reply:
Thanks.